javacord-api/src/main/java/org/javacord/api/DiscordApi.java

```diff
@@ -49,6 +49,8 @@
  * This class is the most important class for your bot, containing all important methods, like registering listener.
  */
 public interface DiscordApi extends GloballyAttachableListenerManager {
+    public static final String DISCORD_DOMAIN = "discord.com";
```

Comment:
If it's in the public api, it should have Javadoc comments.
```suggestion

    /*
     * The domain of Discord.
     */
    public static final String DISCORD_DOMAIN = "discord.com";
```